main.go

```diff
@@ -132,16 +216,31 @@ func caduceus(arguments []string) int {
 		modifiedWRPCount:         metricsRegistry.NewCounter(ModifiedWRPCounter),
 		maxOutstanding:           0,
 	}
+
 	caduceusConfig.Webhook.Logger = logger
 	caduceusConfig.Webhook.MetricsProvider = metricsRegistry
-	svc, stopWatches, err := ancla.Initialize(caduceusConfig.Webhook, caduceusSenderWrapper)
+	argusClientTimeout, err := newArgusClientTimeout(v)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Unable to parse argus client timeout config values: %s \n", err.Error())
```

Comment:
```suggestion
		fmt.Fprintf(os.Stderr, "Unable to parse argus client timeout config values: %v\n", err)
```

Suggested extremely minor change to match the other `Fprintf()` statements.

Reply:
Taken care of in latest commit.